CombinatorialGames/Nimber/Simplicity.lean

```diff
@@ -224,6 +216,9 @@ theorem IsGroup.zero : IsGroup 0 where
 theorem IsGroup.one : IsGroup 1 where
   add_lt := by simp
 
+theorem IsGroup.of_le_one {x : Nimber} (h : x ≤ 1) : IsGroup x := by
+  obtain rfl | rfl := Nimber.le_one_iff.1 h <;> simp
```

Comment:
I'm unsure about which one would be better.

```suggestion
  cases Nimber.le_one_iff.1 h <;> simp_all
```

Reply:
I like the second one better.